### DISCUSSION TITLE
Detect Teams / microphone / webcam usage?

### DISCUSSION THREAD

**Discussion by rschuiling** [timmo001/system-bridge#1221] [Ideas] [answered]
Would it be possible for System Bridge to detect an online call in progress? E.g. by detecting when the webcam or microphone is in use?

**Comment by timmo001** [comment, marked as answer]
Camera usage: https://github.com/timmo001/system-bridge/issues/2851 (Can interpret teams from registry key)
Microphone usage: https://github.com/timmo001/system-bridge/issues/2852 (Unable to find solution, please comment there if anyone does find one)
